setuptools/glob.py

```diff
@@ -0,0 +1,169 @@
+"""
+Filename globbing utility. Mostly a copy of `glob` from Python 3.5.
+
+Changes include:
```

Comment:
It is also worth noting that a `diff -U5 /usr/lib/python3.5/glob.py setuptools/glob.py` shows a very nice summary of the changes.